doc/source/wsrep-provider-index.rst

```diff
@@ -451,6 +451,47 @@ the :file:`galera.cache` file which is used as source for |IST|. The bigger the
 value of this variable, the better are chances that the re-joining node will
 get IST instead of |SST|.
 
+.. variable:: gcomm.thread_prio
+
+Raises the gcomm thread priority to a higher level. Use this variable when the cluster does not receive enough CPU time and competes with other threads. For example, if the threads are not frequently run, a node may drop from the cluster because of the timeout.
```

Comment:
What I feel is, this line doesn't clearly tell what competes with other threads. So, I would prefer.
```suggestion
Raises the gcomm thread priority to a higher level. Use this variable when the gcomm thread does not receive enough CPU time due to other competing threads. For example, if the gcomm threads are not frequently run, a node may drop from the cluster because of the timeout.
```